scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowFunctions.scala

```diff
@@ -449,3 +449,23 @@ final class TFRecordSCollectionFunctions[T <: Array[Byte]](private val self: SCo
   }
 
 }
+
+final class TFSequenceExampleSCollectionFunctions[T <: SequenceExample](
+  private val self: SCollection[T])
+    extends AnyVal {
+
+  /**
+   * Saves this SCollection of `org.tensorflow.example.SequenceExample` as a TensorFlow
+   * TFRecord file.
+   *
+   * @return
+   */
+  def saveAsTfSequenceExampleFile(
```

Comment:
this is in line with the rest of the stuff but I'm getting the idea that we probably could rename this methods to just `saveAsTfRecordFile`. This would be on another PR but @martinbomio  wdyt?

Reply:
+1

Reply:
sounds good to me!

Reply:
created https://github.com/spotify/scio/pull/1762